config/config.exs

```diff
@@ -25,10 +25,11 @@ config :mime, :types, %{
 
 config :livebook,
   teams_url: "https://teams.livebook.dev",
+  agent_name: nil,
   app_service_name: nil,
   app_service_url: nil,
   authentication_mode: :token,
-  feature_flags: [],
+  feature_flags: [deployment_groups: true],
```

Comment:
Please rebase. This branch should not touch feature flags since those changes are already in main. :D